public/assets/bootstrap/js/bootstable.js

```diff
@@ -208,6 +208,7 @@ function butRowEdit(but) {
     //Start the edition mode for a row.
     var $row = $(but).parents('tr');  //accede a la fila
     var $cols = $row.find('td');  //lee campos
+    var $category = $row.find('td:eq(3)').html();
```

Comment:
can 3 be idx

`var $category = $row.find('td:eq(3)').html();`
`var $category = $row.find('td:eq(idx)').html();`

Reply:
var $category can be removed. The var cont contains what you need to do the compare.